### PR TITLE
Galaxy sequence utils

### PR DESCRIPTION
**Pull request by OlgaErm**
Default Galaxy tools from tools-devteam/tool_collections/galaxy_sequence_utils are not working since 'galaxy_sequence_utils-1.0.0' doesn't exist as conda recipe.

### CODE REVIEW

**Comment by bgruening** [comment]
@OlgaErm thanks a lot for the fix. Could you please also bump the tool version.
To help other with the older version and if you have time you also good add the 1.0.0 version to bioconda. I will help you if you have problems.

**Comment by mvdbeek** [comment]
That's more or less a duplicate of https://github.com/galaxyproject/tools-devteam/pull/373, I think.

**Comment by OlgaErm** [comment]
@bgruening there was no consistency in tool versions. I just added +0.0.1 to all of the changed tools. I'll ask @Eugene79 first if he can help me with 1.0.0 Conda recipe. ))) 

Question:
There are 1.1.0 conda recipes for this package, but they are specific for 2. and 3. python versions. If we add just
 " requirement type="package" version="1.1.0">galaxy_sequence_utils</requirement "
into .xml, it starts download version for python3 (it chooses first from the list, I think). How to force conda to download the version for python2 instead?

**Comment by bgruening** [comment]
is there anything wrong with the python3 version?
In case you did not specify a python version it will download the python version that is installed in your conda-environment. Assuming you have miniconda3 (Galaxy default), that you do have a python3 environment and conda will get you python3 packages.

```
"<requirement type="package" version="2.7">python</requirement>" 
```

to the mix will get you python2 packages.

**Comment by OlgaErm** [comment]
`"<requirement type="package" version="2.7">python</requirement>" `
didn't help, in this case we still downloading  linux-64/galaxy_sequence_utils-1.1.0-py36_0.tar.bz2. 

The problems with Python3 are print "..." instead of  print ("...") and execfile instead of exec

**Comment by OlgaErm** [comment]
@bgruening, you were perfectly right, in planemo adding of python 2.7 package is working, it is some problem with our environment. 
Thank you!

File: tool_collections/galaxy_sequence_utils/fastq_combiner/fastq_combiner.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<tool id="fastq_combiner" name="Combine FASTA and QUAL" version="1.0.1">
+<tool id="fastq_combiner" name="Combine FASTA and QUAL" version="1.0.2">
   <description>into FASTQ</description>
   <requirements>
-    <requirement type="package" version="1.0.0">galaxy_sequence_utils</requirement>
+    <requirement type="package" version="1.0.1">galaxy_sequence_utils</requirement>
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_combiner/fastq_combiner.xml]: Why not upgrade to the latest version 1.1.0?

**Comment by nsoranzo** [comment]
@OlgaErm I'm closing this pull request in favour of https://github.com/galaxyproject/tools-devteam/pull/449 , which contains the same changes and more, hopefully we will merge that soon.

**Pull request closed without merging**